docs/framework/release-notes/2025/07-08-july-cumulative-update.md

```diff
@@ -0,0 +1,87 @@
+---
+title: July 2025 security and quality rollup
+description: Learn about the improvements in the .NET Framework July 2025 security and quality rollup.
+ms.date: 07/08/2025
+---
+# July 2025 security and quality rollup
+
+_Released July 8, 2025_
+
+## Summary of what's new in this release
+
+- [Security improvements](#security-improvements)
+- [Quality and reliability improvements](#quality-and-reliability-improvements)
+
+### Security improvements
+
+There are no new security improvements in this release. This update is cumulative and contains all previously released security improvements.
+
+### Quality and reliability improvements
+
+This release contains the following quality and reliability improvements.
+
+#### .NET Libraries
+
+Addresses an issue with concurrency in TypeDescriptor.GetConverter()issue in TypeDescriptor GetProperties() and GetConverter(). (*Applies to: .NET Framework 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8, 4.8.1.*)
```

Comment:
this description needs some work. I don't think it's just a matter of adding the missing space after the first GetConverter()